tools/no-bad-latin.py

```diff
@@ -12,13 +12,17 @@
 from pull_files import filter_files
 
 HERE = os.getcwd()
```

Comment:
this variable is no longer used anywhere

```suggestion
```

Reply:
THANKS! removed, force-pushed